### PR TITLE
Re-Add Reporter Email Form to /admin/reports

### PR DESCRIPTION
**Pull request by juliannatetreault**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
This PR re-adds the ability to manually send emails to abuse reporters via `/admin/reports` and adds a new, informative banner to the Reporter Email form that informs Admins of the following: `"Please note that all users who report abuse receive an automatic confirmation email. You can send an additional, follow-up email to this user by using the form below."`. For additional context, here is what needs to be implemented for this work to be considered "done," as stated in the issue for this PR:

> We should bring back that functionality that was removed previously since Forem admins occasionally need to be able to send a manual email in addition to the automatic ones.
>
> Additionally, for full clarity and communication, it seems important to include a note on this page that tells the Forem Admin that the user already received a confirmation email (sent automatically).

## Related Tickets & Documents
This PR closes issue [395](https://github.com/forem/internalEngineering/issues/395) and relates to PR #12567 

## QA Instructions, Screenshots, Recordings
To QA this PR, please first ensure that you have an `admin`-based role, then do the following:
- Navigate to `/admin/reports` and observe that the "Reporter" email form has been added back, with a `crayons` info banner that informs Admins of the following: `"Please note that all users who report abuse receive an automatic confirmation email. You can send an additional, follow-up email to this user by using the form below."`:
Screen Shot 2021-03-16 at 12 46 53 PM
- While here, test that you are able to send a Reporter an email manually by clicking the "Send Email" button, observing a confirmation that the email was sent, and verifying that the email was sent by checking `/admin/reports`, your console, or your `development.log` for the email:

**_Local Confirmation_**:

Screen Shot 2021-03-16 at 12 49 53 PM
Screen Shot 2021-03-16 at 4 18 36 PM

**_Email Confirmation in Development Log_**:
```
Date: Tue, 16 Mar 2021 12:49:37 -0600
From: DEV Community <yo@dev.to> (local)
To: ullrich+charles@ratke.org
Subject: DEV(local) Community Report

----==_mimepart_6050fdc14d4f4_344c42860712f0
Content-Type: text/plain;
 charset=UTF-8
Content-Transfer-Encoding: 7bit


Hi there,
Thank you for flagging content that may be in violation of the DEV(local) Community Code of Conduct and/or our Terms of Use. We are looking into your report and will take appropriate action.
We appreciate your help as we work to foster a positive and inclusive environment for all!
DEV(local) Community Team



----==_mimepart_6050fdc14d4f4_344c42860712f0
```

### UI accessibility concerns?
Not that I know of!

## Added tests?

- [x] Yes
- [ ] No, and this is why: _please replace this line with details on why tests
      have not been included_
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [x] I've updated the [Developer Docs](https://docs.forem.com) and/or
      [Admin Guide](https://forem.gitbook.io/forem-admin-guide/), or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
🔍   **_You can find the updated Admin Guide here_**: https://forem.gitbook.io/forem-admin-guide/admin/reports
- [ ] I've updated the README or added inline documentation
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [x] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [ ] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] Are there any post-deployment tasks we need to perform?
N/A

## [optional] What gif best describes this PR or how it makes you feel?

Reporter Dog Without a Comment

### CODE REVIEW

File: spec/helpers/feedback_messages_helper_spec.rb [hunk 1]
```diff
@@ -10,6 +10,15 @@
     end
   end
 
+  describe "#reporter_email_details" do
+    it "have proper subject and body" do
```
Comment (vaidehijoshi) [inline spec/helpers/feedback_messages_helper_spec.rb:14]:
```suggestion
    it "has proper subject and body" do
```
The world's tiniest nitpick but I think we want this to read as "it has the proper subject and body" :)

**Review by vaidehijoshi — APPROVED** [review]

File: spec/helpers/feedback_messages_helper_spec.rb (hunk 1 above)
Comment (juliannatetreault) [reply spec/helpers/feedback_messages_helper_spec.rb:14]: Oops! I copied and pasted this without even noticing... great catch!! 😊

File: app/helpers/feedback_messages_helper.rb [hunk 2]
```diff
@@ -15,6 +15,17 @@ def offender_email_details
     { subject: "#{SiteConfig.community_name} Code of Conduct Violation", body: body }.freeze
   end
 
+  def reporter_email_details
+    body = <<~HEREDOC
```
Comment (citizen428) [inline app/helpers/feedback_messages_helper.rb:19]: Suggestion: in some other places we assign the template string to a constant with placeholders and then use `format` to fill in the values, e.g.

```ruby
REPORTER_EMAIL_BODY = <<~HEREDOC
  Hi there,
  Thank you for flagging content that may be in violation of the %<community>s Code of Conduct and/or our Terms of Use. We are looking into your report and will take appropriate action.
  We appreciate your help as we work to foster a positive and inclusive environment for all!
  %<community>s Team
HEREDOC

def reporter_email_details
  body = format(REPORTER_EMAIL_BODY, community: SiteConfig.community_name)
  subject: "#{SiteConfig.community_name} Report", body: body }.freeze
end
```

Comment (juliannatetreault) [reply app/helpers/feedback_messages_helper.rb]: Interesting. Updated per your suggestion! Additionally, I addressed some inconsistencies with possessive/non-possessive proper nouns within all three of the email bodies. Let me know what you think. 😊

**Review by joshpuetz — APPROVED** [review]

**Pull request merged**